content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -154,4 +154,8 @@ the *dataSource* field in the `PersistentVolumeClaim` object.
 For more details, see
 [Volume Snapshot and Restore Volume from Snapshot](/docs/concepts/storage/persistent-volumes/#volume-snapshot-and-restore-volume-from-snapshot-support).
 
+## Use Cases for This Feature
```

Comment:
I think it's nice to put the use cases at the top, before diving into the detailed api.

It may be good to understand how we distinguish this use case from volume cloning.

cc @xing-yang

Reply:
These are great points! I went ahead and moved the use cases into a subsection of the intro. I also added language to implicitly differentiate this from volume cloning.